pages.es/linux/usermod.md

```diff
@@ -1,12 +1,21 @@
 # usermod
 
 > Modifica una cuenta de usuario.
+> Ver también: `users`, `useradd`, `userdel`.
```

Comment:
```suggestion
> Vea también: `users`, `useradd`, `userdel`.
```

Reply:
In other translations it seemed that imperative was avoided.  Maybe we can continue with this line of thought?

El imperativo es un poco más regional, cuando se usa con usted o con tu, por ejemplo vea o ve (ambos del verbo ver). Si lo hacemos en impersonal, tal vez es menos regionalizado en la preferencia al tutear. Sin embargo, puede ser una buena idea tener unos lineamientos en algún sitio específico para español,  tal vez me faltó mirar un poco más de documentación y seguir lo que ya se ha hecho previamente.  

Yo continúo como venga la línea :)